pkg/receive/otlptranslator/histograms.go

```diff
@@ -0,0 +1,219 @@
+// Copyright (c) The Thanos Authors.
+// Licensed under the Apache License 2.0.
+
+// Provenance-includes-location: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/95e8f8fdc2a9dc87230406c9a3cf02be4fd68bea/pkg/translator/prometheusremotewrite/histograms.go
+// Provenance-includes-license: Apache-2.0
+// Provenance-includes-copyright: Copyright The OpenTelemetry Authors.
```

Comment:
Just out of curiosity, what is the plan on maintaining those files? Will we have some kind of script that will map/download them from the source repos? 

Also, sorry the lazyness to not look into it, but why can't we use those as libraries?